src/uiveri5.js

```diff
@@ -310,6 +312,41 @@ function run(config) {
           });
       };
 
+      browser.executeScriptHandleErrors = function executeScriptHandleErrors(scriptName,params) {
+        var code = clientsidescripts[scriptName];
+        params = params || {};
+        browser.controlFlow().execute(function () {
+          logger.debug('Execute script: ' + scriptName + ' with params: ' + JSON.stringify(params));
+          logger.trace('Execute script code: \n' + JSON.stringify(code));
+        });
+        return browser.executeScript(code,params)
+          .then(function (res) {
+            if (res.log) {
+              logger.debug('Script: ' + scriptName + ' logs: \n' + res.log);
+            }
+            if (res.error) {
+              logger.debug('Script: ' + scriptName + ' error: ' + JSON.stringify(res.error));
+              throw new Error(scriptName + ': ' + res.error);
+            }
+            logger.debug('Script: ' + scriptName + ' result: ' + JSON.stringify(res.value));
+            return res.value;
+          });
+      };
+
+      browser.executeScriptLogResult = function executeScriptLogResult(scriptName,params) {
+        var code = clientsidescripts[scriptName];
+        params = params || {};
+        browser.controlFlow().execute(function () {
+          logger.debug('Execute script: ' + scriptName + ' with params: ' + JSON.stringify(params));
+          logger.trace('Execute script code: \n' + JSON.stringify(code));
```

Comment:
i'd rather log the name as it was before

Reply:
scriptName is logged in a similar way, we just avoid the duplication in the script name and refefence ?